### PR TITLE
enh(mc): refresh cloud-azure-database-sqldatabase

### PR DESCRIPTION
**Pull request by omercier**
## Description

Refreshed the procedure.

## Target version (i.e. version that this PR changes)

- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Le connecteur apporte les modèles de service suivants
 </TabItem>
 <TabItem value="Non rattachés à un modèle d'hôte" label="Non rattachés à un modèle d'hôte">
 
-| Alias         | Modèle de service                              | Description                                     |
+| Alias         | Modèle de service                                     | Description                                     |
 |:--------------|:------------------------------------------------------|:------------------------------------------------|
 | App-Resources | Cloud-Azure-Database-SqlDatabase-App-Resources-custom | Contrôle les métriques CPU et Mémoire de l'App  |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md:35]:
```suggestion
| App-Resources | Cloud-Azure-Database-SqlDatabase-App-Resources-custom | Contrôle les métriques CPU et mémoire de l'app  |
```

Comment (cg-tw) [reply i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md:35]: de l'application?

Comment (omercier) [reply i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md]: la contraction semble assumée et sans la majuscule https://azure.microsoft.com/en-us/products/azure-sql/database/

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md [hunk 2]
```diff
@@ -390,28 +393,25 @@ que le connecteur arrive bien à superviser une instance Azure en utilisant une
 telle que celle-ci (remplacez les valeurs d'exemple par les vôtres) :
 
 ```bash
-/usr/lib/centreon/plugins//centreon_azure_database_sqldatabase_api.pl \
+/usr/lib/centreon/plugins/centreon_azure_database_sqldatabase_api.pl \
 	--plugin=cloud::azure::database::sqldatabase::plugin \
-	--mode=app-resources \
+	--mode=workers \
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md:398]: not many parameters here, is that normal? the mandatory ones should be filled in, at least

File: pp/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md [hunk 3]
```diff
@@ -236,31 +236,34 @@ yum install centreon-plugin-Cloud-Azure-Database-SqlDatabase-Api
...
+| AZURESQLSERVERNAME | Set resource name or ID. It is the database name.                                                                      |                   | X           |
+| AZURESUBSCRIPTION  | Set Azure subscription (required if logged to several subscriptions)                                                       |                   | X           |
+| AZURETENANT        | Set Azure tenant ID                                                                                                        |                   | X           |
+| PROXYURL           | Proxy URL if any                                                                                                           |                   |             |
+| EXTRAOPTIONS       | Any extra option you may want to add to every command (a --verbose flag for example). All options are listed [here](#available-options).                       |                   |             |
+
+> Set the following options in the EXTRAOPTIONS macro if you are monitoring resource from Microsoft Azure operated by 21Vianet (Azure China):
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md:252]:
```suggestion
> Set the following options in the EXTRAOPTIONS macro if you are monitoring resources from Microsoft Azure operated by 21Vianet (Azure China):
```

File: pp/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md [hunk 4]
```diff
@@ -388,28 +391,25 @@ Once the plugin is installed, log into your Centreon poller's CLI using the
 is able to monitor an Azure Instance using a command like this one (replace the sample values by yours):
 
 ```bash
-/usr/lib/centreon/plugins//centreon_azure_database_sqldatabase_api.pl \
+/usr/lib/centreon/plugins/centreon_azure_database_sqldatabase_api.pl \
 	--plugin=cloud::azure::database::sqldatabase::plugin \
-	--mode=app-resources \
+	--mode=workers \
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md:396]: same as for FR

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md [hunk 5]
```diff
@@ -390,28 +393,25 @@ que le connecteur arrive bien à superviser une instance Azure en utilisant une
...
-	--subscription='xxxxxxxxx' \
-	--tenant='xxxxxxxxx' \
-	--client-id='xxxxxxxxx' \
-	--client-secret='xxxxxxxxx' \
+	--resource='' \
+	--resource-group='' \
+	--subscription='' \
+	--tenant='' \
+	--client-id='' \
+	--client-secret='' \
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md:405]:
```suggestion
	--resource='XXXXXXXXXXXXXXXX' \
	--resource-group='' \
	--subscription='XXXXXXXXXXXXXXXX' \
	--tenant='XXXXXXXXXXXXXXXX' \
	--client-id='XXXXXXXXXXXXXXXX' \
	--client-secret='XXXXXXXXXXXXXXXX' \
```

File: pp/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md [hunk 6]
```diff
@@ -388,28 +391,25 @@ Once the plugin is installed, log into your Centreon poller's CLI using the
...
-	--subscription='xxxxxxxxx' \
-	--tenant='xxxxxxxxx' \
-	--client-id='xxxxxxxxx' \
-	--client-secret='xxxxxxxxx' \
+	--resource='' \
+	--resource-group='' \
+	--subscription='' \
+	--tenant='' \
+	--client-id='' \
+	--client-secret='' \
```
Comment (omercier) [inline pp/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md:403]:
```suggestion
	--resource='XXXXXXXXXXXXXXXX' \
	--resource-group='' \
	--subscription='XXXXXXXXXXXXXXXX' \
	--tenant='XXXXXXXXXXXXXXXX' \
	--client-id='XXXXXXXXXXXXXXXX' \
	--client-secret='XXXXXXXXXXXXXXXX' \
```

**Review by cg-tw — APPROVED** [review]

**Pull request merged**